backbone/src/main/java/org/researchstack/backbone/model/survey/NavigationStep.java

```diff
@@ -0,0 +1,24 @@
+package org.researchstack.backbone.model.survey;
+
+import org.researchstack.backbone.result.StepResult;
+import org.researchstack.backbone.result.TaskResult;
+import org.researchstack.backbone.step.QuestionStep;
+
+import java.util.List;
+
+/**
+ * Created by TheMDP on 12/31/16.
+ */
+
+public interface NavigationStep {
```

Comment:
Do the classes using NavigationStep need to set these values, or just retrieve them? I'm thinking maybe some NavigationSteps have skip logic that is more complicated, do you know if that's the case?